db/schema.rb

```diff
@@ -187,14 +199,16 @@
   end
 
   create_table "messages", :force => true do |t|
-    t.string   "subject"
+    t.integer  "recipient_id"
     t.string   "status"
-    t.string   "body"
-    t.string   "channels"
-    t.datetime "sent_at"
-    t.integer  "created_by"
-    t.datetime "created_at", :null => false
-    t.datetime "updated_at", :null => false
+    t.string   "channel"
+    t.datetime "processed_at"
+    t.string   "processed_by"
+    t.string   "slug"
+    t.datetime "created_at",   :null => false
+    t.datetime "updated_at",   :null => false
+    t.text     "body"
+    t.string   "subject"
```

Comment:
Unfortunately these changes to "messages" are regressions, as are the reintroduction of the "notifications", "recipients", and "responses" below.  Also, below, two "resource type" tables are introduced.  We want the "resource_types" one.

Fortunately, we already investigated & solved this problem in #81, so we can fix this again.  We Have The Technology.  :runner: